.travis.yml

```diff
@@ -14,19 +14,19 @@ go:
 
 # A build matix defines the K8s versions to use for e2e tests. Travis runs these in parrellel
 env:
-  - KIND_K8S_VERSION="v1.16.2"
+  - KIND_K8S_VERSION="v1.16.4"
```

Comment:
[`kind`](https://kind.sigs.k8s.io/) doesn't all of the k8s versions.
Looking at https://hub.docker.com/r/kindest/node/tags, the latest supported 1.16 is 1.16.3.
You can either change it to 1.16.3 or leave it as is.

Reply:
Fixed!